Lib/test/test_dict.py

```diff
@@ -37,6 +37,36 @@ def test_literal_constructor(self):
             dictliteral = '{' + ', '.join(formatted_items) + '}'
             self.assertEqual(eval(dictliteral), dict(items))
 
+    def test_pep_584(self):
```

Comment:
Maybe rename to `test_or_operator` or `test_merge_operator`?

Reply:
Since the PEP refers to it as the "merge" operator, I think that would be more descriptive for the test name in comparison to "or" (despite the symbol itself being more commonly known as the "or" operator).

Reply:
One reason I originally named it this is that the name is self-documenting. The purpose of the test is clear, and it makes it simple for a future maintainer to find a resource outlining the desired behavior. I'm fine changing it though.